js/containers/App.jsx

```diff
@@ -120,11 +108,22 @@ function App(props) {
 
 const mapStateToProps = function(state) {
 	return {
-		message: state.message,
-		messageType: state.message_type,
-		activeStep: state.activeStep,
+		isLoading: [
+			state.git.is_loading || state.git.is_updating,
+			state.plan.is_loading,
+			false,
+			false
+		],
+		isFinished: [
+			state.git.selected_ref !== "",
+			state.plan.validation_code === 'success',
+			state.plan.validation_code === 'success' && (state.approval.approved || state.approval.bypassed),
```

Comment:
skip the whole approval process, go straight to start, collect $200 and a deploy